dev_guide/templates.adoc

```diff
@@ -105,13 +86,77 @@ $ osc process -f examples/sample-app/application-template-dockerbuild.json -v AD
 
 ====
 
-The JSON string can either be redirected to a file or applied directly by piping the process output to the `apply` command, as shown in the following example.
+The JSON string can either be redirected to a file or applied directly by piping the process output to the `create` command:
 
 .Redirecting JSON String
 ====
 
 ----
-$ osc process -f examples/sample-app/application-template-dockerbuild.json -v ADMIN_USERNAME=root,MYSQL_DATABASE=admin | osc apply -f -
+$ osc process -f examples/sample-app/application-template-dockerbuild.json -v ADMIN_USERNAME=root,MYSQL_DATABASE=admin | osc create -f -
+----
+
+====
+
+=== Creating from Template
+
+To create an configuration from a template using the Management Console:
+
+1. Click on the *Create+* button:
```

Comment:
Maybe mention something about first making sure you're in the right project, or choose the desired project, etc.